stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -447,7 +449,10 @@ extension TensorElementLiteral : ExpressibleByArrayLiteral {
   public typealias ArrayLiteralElement = TensorElementLiteral<Scalar>
   @inlinable @inline(__always)
   public init(arrayLiteral elements: TensorElementLiteral<Scalar>...) {
-    let handle: TensorHandle<Scalar> = #tfop("Pack", elements)
+    // Attr T (non-optional in the op definition) need not be specified when we
```

Comment:
Is this a workaround for your TF patch? Will we be able to eliminate this chance after you pull from TF?